collections.go

```diff
@@ -820,6 +820,22 @@ func handleViewCollection(app *App, w http.ResponseWriter, r *http.Request) erro
 	return err
 }
 
+func handleViewMention(app *App, w http.ResponseWriter, r *http.Request) error {
+	vars := mux.Vars(r)
+	handle := vars["handle"]
+
+	remoteUser, err := app.db.getProfilePageFromHandle(app, handle)
+	if err != nil {
+		log.Error("Couldn't find this user "+handle, err)
+		return nil
+	}
+
+	http.Redirect(w, r, remoteUser, http.StatusSeeOther)
```

Comment:
We should use our common pattern of `return`ing an `impart.HTTPError` instead of calling this method (for example): https://github.com/writeas/writefreely/blob/bc2016f00ff2b2fa58f978f489255457af4f46d5/collections.go#L416

The `handler.Web()` method that wraps this takes care of the actual redirect.

Reply:
Done